app/mailers/devise_mailer.rb

```diff
@@ -5,13 +5,13 @@ class DeviseMailer < Devise::Mailer
   def confirmation_instructions(record, token, opts={})
     @host = CONFIG['checkdesk_base_url']
     @client_host = CONFIG['checkdesk_client']
-    opts[:subject] = I18n.t(:mail_account_confirmation, default: "%{app_name} account confirmation", app_name: CONFIG['app_name'])
+    opts[:subject] = I18n.t(:mail_account_confirmation, default: "%{product_name} account confirmation", product_name: CONFIG['product_name'])
```

Comment:
Note that this will not work because the string is still `%{app_name} account confirmation` on the Transifex side.